doc/modules/changes/20230712_ranpengli

```diff
@@ -0,0 +1,4 @@
+Changed: The entropy plugin now includes a plasticity rheology, which can be set by cohesion and frictional angle. It can also use a viscosity prefactor profile, set the lateral viscosity variation limit, and choose p-T dependent conductivity
+
+<br>
+(Ranpeng Li, 2023/07/12)
```

Comment:
this file is currently missing a newline at the end of the file. This is an annoying default property of VS code, you can check here how to change the behavior: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files